Dockerfile.metering-ansible-operator

```diff
@@ -1,34 +1,27 @@
 # need the helm-cli from the helm image
 FROM quay.io/openshift/origin-metering-helm:latest as helm
 # final image needs kubectl, so we copy `oc` from cli image to use as kubectl.
-FROM openshift/origin-cli:latest as cli
+FROM registry.svc.ci.openshift.org/ocp/4.6:cli as cli
 # the base image is the ansible-operator's origin images
-FROM quay.io/openshift/origin-ansible-operator:4.6
+FROM registry.svc.ci.openshift.org/ocp/4.6:ansible-operator
 
 USER root
-RUN set -x; INSTALL_PKGS="curl bash ca-certificates less which inotify-tools openssl" \
-    && yum clean all && rm -rf /var/cache/yum/* \
-    && yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm  \
-    && yum install --setopt=skip_missing_names_on_install=False -y \
-        $INSTALL_PKGS  \
-    && yum clean all \
-    && rm -rf /var/cache/yum
-
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
+RUN set -x;
+ARG INSTALL_PKGS="less python2-botocore python2-boto3 openssl tini"
+
+# the boto core packages will only install if the following repo are added w/out removing base repos
+RUN curl http://base-4-4.ocp.svc > /etc/yum.repos.d/base-4-4.repo
+RUN yum install --verbose --allowerasing --setopt=skip_missing_names_on_install=False --nobest -y ${INSTALL_PKGS}
+RUN yum clean all
+RUN rm -rf /var/cache/yum
 
 COPY --from=helm /usr/local/bin/helm /usr/local/bin/helm
 COPY --from=cli /usr/bin/oc /usr/bin/oc
-# put tini and kubectl into our path
+# put kubectl into our path
 RUN ln -f -s /usr/bin/oc /usr/bin/kubectl
-RUN ln -f -s /tini /usr/bin/tini
 
-# List of python packages:
-# 1. botocore and boto3 are used by the aws-related modules (aws_s3)
-# 2. netaddr is needed to use the ipv4/ipv6 jinja filter
-# 3. cryptography is used by the openssl_* modules for TLS/authentication purposes
-RUN pip install --no-cache-dir --upgrade ansible~=2.9 openshift botocore boto3 cryptography netaddr
+# netaddr is needed to use the ipv4/ipv6 jinja filter
+RUN pip install --no-cache-dir --upgrade netaddr
```

Comment:
Was unable to find from any combo of base packages a replacement that would install and allow to get rid of the `netaddr` pip install.

Reply:
i assume the OSBS builds aren't doing a pip install (it would fail since it has to reach out remotely), so how are the osbs builds handling this?

Reply:
OSBS: `yum install python-netaddr` but I have so far been unable to locate a public version of that package that is installable with all non-conflicting dependencies.

Reply:
you should be able to get it mirrored into the CI rpm mirror

Reply:
Maybe, but it's not as simple as asking for another package, idt. With the current base and curl'd in base-4.4 the following happens on trying to `yum install python-netaddr` (which is available):
```  
Problem: conflicting requests
  - package python-netaddr-0.7.5-7.el7.noarch requires /usr/bin/python, but none of the providers can be installed
  - package python-netaddr-0.7.5-9.el7.noarch requires /usr/bin/python, but none of the providers can be installed
  - package python-2.7.5-16.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-38.el7_2.x86_64 is filtered out by modular filtering
  - package python-2.7.5-18.el7_1.1.x86_64 is filtered out by modular filtering
  - package python-2.7.5-34.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-39.el7_2.x86_64 is filtered out by modular filtering
  - package python-2.7.5-48.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-58.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-68.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-69.el7_5.x86_64 is filtered out by modular filtering
  - package python-2.7.5-76.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-77.el7_6.x86_64 is filtered out by modular filtering
  - package python-2.7.5-80.el7_6.x86_64 is filtered out by modular filtering
  - package python-2.7.5-79.el7_6.x86_64 is filtered out by modular filtering
  - package python-2.7.5-86.el7.x86_64 is filtered out by modular filtering
  - package python-2.7.5-88.el7.x86_64 is filtered out by modular filtering 
```
I tried all available permutations I think. I wasn't able to tell what was causing the modular filtering or how to avoid/turn it off.

Reply:
ok.  ART might be able to help you sort that out but maybe as a followup.

Reply:
yep as a follow-on or next time we need to touch the Dockerfile